src/k8s/cmd/k8s/k8s.go

```diff
@@ -71,5 +71,6 @@ func NewRootCmd() *cobra.Command {
 	rootCmd.AddCommand(newHelmCmd())
 	rootCmd.AddCommand(newKubectlCmd())
 
+	rootCmd.DisableAutoGenTag = true
```

Comment:
without this, we get this at the bottom of our docs:

`generated by cobra`

which isn't bad, per se.. just not necessary in our reference docs.